packages/core/src/components/Button/Button.scss

```diff
@@ -160,6 +162,10 @@ Inverse buttons
   }
 }
 
+.ds-c-button--transparent-inverse {
```

Comment:
Isn't this already styled above on line 71?

Reply:
Doh!